test/functional/outputs/syslog/forward_to_syslog_test.go

```diff
@@ -122,51 +120,27 @@ var _ = Describe("[Functional][Outputs][Syslog] Functional tests", func() {
 				Expect(framework.WriteMessagesToApplicationLog(log, 1)).To(BeNil())
 			}
 			// Read line from Syslog output
-			outputlogs, err := framework.ReadRawApplicationLogsFrom(logging.OutputTypeSyslog)
+			outputlogs, err := framework.ReadRawApplicationLogsFrom(string(obs.OutputTypeSyslog))
 			Expect(err).To(BeNil(), "Expected no errors reading the logs")
 			Expect(outputlogs).ToNot(BeEmpty())
 			fields := strings.Split(outputlogs[0], " ")
 			Expect(getAppName(fields)).To(Equal("rec_appname"))
 			Expect(getProcID(fields)).To(Equal("rec_procid"))
 			Expect(getMsgID(fields)).To(Equal("rec_msgid"))
 		})
-		It("should take values from fluent tag", func() {
```

Comment:
I would leave this in and just comment the entire block out so we don't forget it